src/components/GameSwitchboard.js

```diff
@@ -1,8 +1,35 @@
+//GameSwitchboard.js made by Jason Iqbal for SweatEquity, a division of sweatyPits.
+
 import React from 'react';
 import House from './House.js';
 import Menu  from './Menu.js';
+import { 
+  Stitch, 
+  RemoteMongoClient,
+  AnonymousCredential } from 'mongodb-stitch-browser-sdk';
 
 class GameSwitchboard extends React.Component {
+    constructor(props) {
+        super(props);
+        
+        this.client = Stitch.initializeDefaultAppClient('sweatequity-cbpxb');
+        this.db = this.client.getServiceClient(RemoteMongoClient.factory, 'mongodb-atlas').db('highScores');
+        
+    }
+  
+    submitScoreInfo = (props) => {
```

Comment:
I don't think props is a good name here. If we have a specific thing we want to pass, we should pass it, but props is for component constructors, not just functions that exist on them.